gossip/src/legacy_contact_info.rs

```diff
@@ -27,8 +27,7 @@ pub struct LegacyContactInfo {
     tvu: SocketAddr,
     /// TVU over QUIC protocol.
     tvu_quic: SocketAddr,
-    /// address to send repair responses to
-    repair: SocketAddr,
+    unused: SocketAddr,
```

Comment:
any reason why this is needed instead of getting fully removed? legacy purposes? -> legacy validators contacting this validator?

Reply:
It won't be backward compatible to remove it because we are using `bincode` for serialization which does not including any tags or schema in the serialized payload.
So if we remove it new code and old code will deserialize the same bytes differently.